core/src/main/scala/com/salesforce/op/stages/impl/feature/OpStringIndexer.scala

```diff
@@ -32,6 +32,7 @@ package com.salesforce.op.stages.impl.feature
 
 import com.salesforce.op.UID
 import com.salesforce.op.features.types._
+import com.salesforce.op.stages.impl.feature.StringIndexerHandleInvalid.{Keep, Skip}
```

Comment:
I think you can do `import com.salesforce.op.stages.impl.feature.StringIndexerHandleInvalid._`
then `assert(Seq(Skip, Error, Keep).contains(value), ...)`